components/resources/commands.grdp

```diff
@@ -0,0 +1,483 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<grit-part>
```

Comment:
Could we have comment saying something like

"Please reuse existing strings as much as possible rather than add new strings here."